pkg/broker/broker.go

```diff
@@ -37,11 +37,12 @@ type Broker interface {
 
 // AnsibleBroker - Broker using ansible and images to interact with oc/kubernetes/etcd
 type AnsibleBroker struct {
-	dao           *dao.Dao
-	log           *logging.Logger
-	clusterConfig apb.ClusterConfig
-	registry      apb.Registry
-	engine        *WorkEngine
+	dao            *dao.Dao
+	log            *logging.Logger
+	clusterConfig  apb.ClusterConfig
+	registry       apb.Registry
+	engine         *WorkEngine
+	lauchApbOnBind bool
```

Comment:
Instead of passing in lauchApbOnBind (which has a typo by the way), it should pass in `brokerConfig BrokerConfig`.